pkg/eks/tasks.go

```diff
@@ -104,6 +125,27 @@ func (c *ClusterProvider) NewTasksRequiringControlPlane(cfg *api.ClusterConfig)
 	return tasks
 }
 
+// ClusterTasksForNodeGroups returns all tasks dependent on node groups
+func (c *ClusterProvider) ClusterTasksForNodeGroups(cfg *api.ClusterConfig, installNeuronDevicePluginParam bool) *manager.TaskTree {
+	tasks := &manager.TaskTree{
+		Parallel:  false,
+		IsSubTask: true,
+	}
+	var reallyInstallNeuronDevicePlugin bool
+	for _, ng := range cfg.NodeGroups {
+		reallyInstallNeuronDevicePlugin = reallyInstallNeuronDevicePlugin || HasInstanceType(ng, utils.IsNeuronInstanceType)
+	}
+	reallyInstallNeuronDevicePlugin = reallyInstallNeuronDevicePlugin && !installNeuronDevicePluginParam
```

Comment:
> `&& !installNeuronDevicePluginParam` 

was this intended or should it be `reallyInstallNeuronDevicePlugin && installNeuronDevicePluginParam`

Reply:
yes it should be! I had it `skipInstall` before. thanks